functions-methods.js

```diff
@@ -20,7 +39,33 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail (email) {
+    if (email.split("@")[1].includes("novi-education.nl")) {
```

Comment:
Als je dezelfde methode vaker gebruikt is het wel weer gebruikelijker om hier een variabele van te maken.